services/datamanager/collection/collection.go

```diff
@@ -131,7 +131,8 @@ func GetCollections(params GetCollectionsParams) (results ReadCollections, err e
 	sdkParams := arvadosclient.Dict{
 		"select":  fieldsWanted,
 		"order":   []string{"modified_at ASC"},
```

Comment:
Even though the API should do it for us automatically, for clarity (and a safety belt) I think we should add "uuid asc" here explicitly -- we're depending on the server to give us a consistent order (among items with identical modified_at) from one page to the next.